features/frameworks/greentea-client/source/greentea_test_env.cpp

```diff
@@ -532,7 +531,7 @@ enum Token {
  * \return Next character from the stream or EOF if stream has ended.
  *
  */
-static int _get_char() {
+extern "C" char greentea_getc() {
```

Comment:
* May it be in `greentea_serial.cpp` ? 
* Is there a reason to not return an `int` like the standard `getc` function ?